frontend/benefit/handler/src/types/application.d.ts

```diff
@@ -154,6 +154,39 @@ export type ApproveTerms = {
   selectedApplicantConsents: string[];
 };
 
+// handler
+
+export interface RowData {
```

Comment:
These seems to be identical to other application type except for the casing, perhaps you could try this: https://stackoverflow.com/questions/60269936/typescript-convert-generic-object-from-snake-to-camel-case
Also I would use `type` instead of `interface` I think that `interface` is better only if supporting object oriented way of doing things.

Reply:
great idea! but i would leave it for separate PR, when we move the common types to shared from applicant and handler. But i ll fix interface to type in this one

Reply:
fixed

Reply:
Ok, it's should not be a big thing to do as typescript would complain if it doesnt work. But let's leave it later.